### PR TITLE
Implemented drawing thick lines.

### PR DESCRIPTION
**Pull request by tmmvigneshm**

### CODE REVIEW

File: Geometry.cs [hunk 1]
```diff
@@ -23,6 +23,15 @@ namespace GrayBMP;
...
 readonly record struct Point2 (double X, double Y) {
    public (int X, int Y) Round () => ((int)(X + 0.5), (int)(Y + 0.5));
+
+   /// <summary>Returns a new point for a given point which is radially rotated.</summary>
+   public readonly Point2 RadialMove (double r, double theta) => new (X + r * Math.Cos (theta), Y + r * Math.Sin (theta));
+
+   /// <summary>Returns angle between two points.</summary>
+   public readonly double AngleTo (Point2 b) {
+      double dx = b.X - X, dy = b.Y - Y;
+      return Math.Atan2 (dy, dx);
```
Comment (tarydon) [inline Geometry.cs]: Try doing a.AngleTo (a)

**Review by tarydon — CHANGES_REQUESTED** [review]

File: Geometry.cs (hunk 1 above)
Comment (tmmvigneshm) [reply Geometry.cs]: Radian value will always zero, so tan inverse computation is not required. Updated the changes.

File: Geometry.cs [hunk 2]
```diff
@@ -30,7 +30,11 @@ namespace GrayBMP;
    /// <summary>Returns angle between two points.</summary>
    public readonly double AngleTo (Point2 b) {
       double dx = b.X - X, dy = b.Y - Y;
-      return Math.Atan2 (dy, dx);
+      if (dx != 0 || dy != 0) {
+         var fAngle = Math.Atan2 (dy, dx);
+         return fAngle.EQ (-Math.PI) ? Math.PI : fAngle;
```
Comment (tarydon) [inline Geometry.cs:35]: Why this special treatment for -PI ?

**Review by tarydon — CHANGES_REQUESTED** [review]

File: Geometry.cs (hunk 2 above)
Comment (tmmvigneshm) [reply Geometry.cs:35]: When delta Y is negative zero (applicable for floating point numbers), we might get -180 degrees. Although this piece of code is not applicable in this case, I just added this for completeness of the **AngleTo** method.

**Pull request closed without merging**